src/App.tsx

```diff
@@ -1,24 +1,28 @@
 import { Route, BrowserRouter as Router, Routes } from 'react-router-dom'
 import { ToastContainer } from 'react-toastify'
 import { lazy } from 'react'
-import { space } from 'classnames/tailwind'
 import Announcement from 'components/Announcement'
 import Footer from 'components/Footer'
 import LazyComponent from 'components/LazyComponent'
 import Navbar from 'components/navbar/Navbar'
 import Root from 'components/Root'
+import classnames, { overflow, space } from 'classnames/tailwind'
 
 const NotFound = lazy(() => import('pages/NotFound'))
 const OwnedBadge = lazy(() => import('pages/OwnedBadge'))
 const Landing = lazy(() => import('pages/Landing'))
 const Main = lazy(() => import('pages/Main'))
+const pageContainer = classnames(
+  space('space-y-6', 'sm:space-y-10'),
+  overflow('overflow-x-hidden')
```

Comment:
Usually we can avoid adding `x-hidden`, if we do so, we must check literally everything, because it can break many things